core/services/ocr2/plugins/functions/integration_tests/v1/internal/testutils.go

```diff
@@ -145,8 +144,6 @@ func CreateAndFundSubscriptions(t *testing.T, owner *bind.TransactOpts, linkToke
 	_, err = linkToken.TransferAndCall(owner, routerContractAddress, amount, data)
 	require.NoError(t, err)
 
-	time.Sleep(1000 * time.Millisecond)
```

Comment:
Why not wait for the commit here too instead of having to wait for the commit after this function returns?

Reply:
Yeah we could, for consistency. I'll change it.